### ISSUE TITLE
Suggestion: Command to change portal appearance

### ISSUE THREAD

**Issue by Omanoctoa** [Multiverse/Multiverse-Portals#474]
Hello!

Firstly, love this plugin. MV Portals makes navigation and transportation so much more enjoyable.

One suggestion to improve it: I love being able to make portals out of different materials, though it seems to have issues with generic 'portal' blocks which often face the wrong direction. Apart from solving that issue...I would love the ability to change a portal's portal-block (water/fire/portal) remotely via command. Is this possible?

Basically, the ability to make a portal appear to turn on or off, or to change from say, lava to water. I think this would be pretty cool :D

**Comment by Jadiction** [comment]
This can already be done. MV-Portals can be used on any block. It just sets regions to teleports, so you could make a cube with water and set a portal to it.
